packages/core/lib/components/CriticalAlert.tsx

```diff
@@ -0,0 +1,235 @@
+import { Label } from './Label';
+import { LinkButton } from './LinkButton';
+
+type AlertVariant = 'danger' | 'ok' | 'info';
+
+export type CriticalAlertProps = {
+  text: string;
+  variant: AlertVariant;
+  link?: string;
+  title?: string;
+  className?: string;
+};
+
+//TODO: need i18n
+const ALERT_TAG_MAP: Record<AlertVariant, string> = {
+  danger: '긴급',
+  ok: '안전',
+  info: '안내',
+} as const;
+
+const variantStyles: Record<
+  NonNullable<CriticalAlertProps['variant']>,
+  {
+    fill: { backgroundColor: string; color: Color };
+  }
+> = {
+  info: {
+    fill: { backgroundColor: colors.primary70, color: 'gray-90' },
+  },
+  ok: {
+    fill: { backgroundColor: colors.success, color: 'success-70' },
+  },
+  danger: {
+    fill: { backgroundColor: colors.danger, color: 'danger-70' },
+  },
+};
+
+export const CriticalAlert = ({
+  text,
+  variant,
+  link,
+  title,
+  className,
+}: CriticalAlertProps) => {
+  return (
+    <div
+      className={`
+            container
+            mx-auto
+            border
+            border-gray-0
+            rounded-2
+            bg-gray-0
+            px-5
+            py-3 
+            ${className}
+        `}
+    >
+      <div
+        className={`
+            flex
+            justify-between            
+            items-center 
+            gap-3
+        `}
+      >
+        <UrgentBadge variant={variant} />
+        <Label className="flex-auto" color="gray-100" size="m" weight="bold">
+          {text}
+        </Label>
+        <div className="hidden md:block">
+          {link && <MoreLinkButton link={link} title={title} />}
+        </div>
+        <div className="block md:hidden">
+          {link && (
+            <LinkButton
+              title="자세히보기"
+              link={link}
+              style={{
+                textDecoration: 'none',
+              }}
+            >
+              <ArrowIcon />
+            </LinkButton>
+          )}
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const UrgentBadge = ({ variant }: { variant: AlertVariant }) => {
+  const { backgroundColor } = variantStyles[variant].fill;
+  const tag = ALERT_TAG_MAP[variant];
+
+  return (
+    <span
+      className={`
+            inline-flex
+            items-center
+            justify-center
+            rounded-full
+            gap-1      
+        `}
+      style={{
+        backgroundColor,
+        width: '78px',
+        height: '48px',
+      }}
+    >
+      {variant === 'info' && <InfoIcon />}
+      {variant === 'ok' && <OkIcon />}
+      {variant === 'danger' && <DangerIcon />}
+      <Label color="gray-0" size="s" weight="bold">
+        {tag}
+      </Label>
+    </span>
+  );
+};
+
+const MoreLinkButton = ({ link, title }: { link: string; title?: string }) => {
+  return (
+    <LinkButton
+      link={link}
+      title={title}
+      style={{
+        textDecoration: 'none',
+      }}
+    >
+      <Label color="gray-100" size="m" className="hidden md:inline-block">
```

Comment:
Could you add `cursor-pointer` in the className? Currently `Label` component remove the cursor css setting, so we need to manually add this :(